Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -366,13 +370,13 @@ class CustomBuildTaskTerminal implements Pseudoterminal {
                     if (_error) {
                         telemetry.logLanguageServerEvent("cppBuildTaskError");
                         const dot: string = (stdout || _stderr) ? ":" : ".";
-                        this.writeEmitter.fire(`Build finished with error${dot}${this.endOfLine}`);
+                        this.writeEmitter.fire(localize("build_finished_with_error", "Build finished with error") + dot + this.endOfLine);
```

Comment:
"Build finished with error." is not grammatically correct...if that case is possible it should be "Build finished with an error" or "Build finished with errors", so correct translation of "Build finished with error" might not have enough info to translate correctly...something like "Build finished with errors:" might work.

Reply:
the `dot `variable is either `:` or `.` that is why it is not included in the sentence.
I replaced it with a more common sentence `Build finished with error(s)`